src/locale/languages/es.js

```diff
@@ -0,0 +1,338 @@
+      openButton: 'Abiertas',
+      closedButton: 'Cerradas',
+      searchingMessage: 'Buscando {{query}}',
+      noOpenIssues: 'No se encontraron incidencias abiertas!',
+      noClosedIssues: 'No se encontraron incidencias cerradas!',
+    },
+    pullList: {
+      title: 'Pull Requests',
+      openButton: 'Abiertos',
+      closedButton: 'Cerrados',
+      searchingMessage: 'Buscando {{query}}',
+      noOpenPulls: 'No se encontraron pull requests abiertos!',
+      noClosedPulls: 'No se encontraron pull requests cerrados!',
+    },
+    pullDiff: {
+      title: 'Diferencias',
+      numFilesChanged: '{{numFilesChanged}} archivos',
+      new: 'NUEVOS',
+      deleted: 'ELIMINADOS',
+      fileRenamed: 'Archivo renombrado sin cambios',
+    },
+  },
+  organization: {
+    main: {
+      membersTitle: 'MIEMBROS',
+      descriptionTitle: 'DESCRIPCIÓN',
+    },
+  },
+  issue: {
+    settings: {
+      title: 'Configuración',
+      pullRequestType: 'Pull Request',
+      issueType: 'Incidencias',
+      applyLabelButton: 'Aplicar etiqueta',
+      noneMessage: 'Ninguno todavía',
+      labelsTitle: 'ETIQUETAS',
+      assignYourselfButton: 'Asígnate a ti mismo',
+      assigneesTitle: 'ASIGNADOS',
+      actionsTitle: 'ACCIONES',
+      unlockIssue: 'Desbloquear {{issueType}}',
+      lockIssue: 'Bloquear {{issueType}}',
+      closeIssue: 'Cerrar {{issueType}}',
+      reopenIssue: 'Reabrir {{issueType}}',
+      areYouSurePrompt: '¿Estás seguro?',
+      applyLabelTitle: 'Asignar etiqueta a esta incidencia',
+    },
+    comment: {
+      commentActions: 'Acciones de comentarios',
+      editCommentTitle: 'Editar Comentario',
+      editAction: 'Editar',
+      deleteAction: 'Eliminar',
+    },
+    main: {
+      assignees: 'Assignees',
+      mergeButton: 'Fusionar Pull Request',
+      noDescription: 'No se ha proporcionado ninguna descripción.',
+      lockedCommentInput: 'Bloqueado, pero todavía puedes comentar...',
+      commentInput: 'Añade un comentario...',
+      lockedIssue: 'La incidencia está bloqueada',
+      states: {
+        open: 'Abierta',
+        closed: 'Cerrada',
+        merged: 'Fusionada',
+      },
+      screenTitles: {
+        issue: 'Incidencia',
+        pullRequest: 'Pull Request',
+      },
+      openIssueSubTitle: '#{{number}} abiertas hace {{time}} por {{user}}',
+      closedIssueSubTitle: '#{{number}} por {{user}} fue cerrada hace {{time}}',
+    },
+    newIssue: {
+      title: 'Nueva incidencia',
+      missingTitleAlert: 'Debes especificar un título para la incidencia!',
+      issueTitle: 'Título de la incidencia',
+      writeATitle: 'Escribe un título para la incidencia aquí',
+      issueComment: 'Comentario de la incidencia',
+      writeAComment: 'Escribe un comentario para tu incidencia aquí',
+    },
+    pullMerge: {
+      title: 'Fusionar Pull Request',
+      createMergeCommit: 'Create a merge commit',
+      squashAndMerge: 'Squash y Merge',
+      merge: 'fusionar',
```

Comment:
I think we should leave `squash` and `merge` and any git reserved word in English. What you guys think @lex111 @khvilaboa @andrewda @housseindjirdeh ?

Reply:
@alejandronanez it all depends on how you decided to translate in your language these phrases..

For example, you can see how this is done in the [book on Git, the Spanish version](https://git-scm.com/book/es/v1/Ramificaciones-en-Git-Procedimientos-b%C3%A1sicos-para-ramificar-y-fusionar).

Reply:
@alejandronanez I commited a version with the git reserved words without translate (I have seen that other translations follow this pattern).